mahjong/src/App.js

```diff
@@ -20,6 +17,7 @@ function App() {
   }, []);
 
   const onTurnCard = (number, index) => () => {
+    if(currentIndex.length === 2) return
```

Comment:
Avoid using one liner `if`s

Also, be consistent with spaces for code blocks. Choose either `if (...) {}` or `if(...) {}`